ECLIPs/ECIP-draft_succinct-proofs.md

```diff
@@ -0,0 +1,142 @@
+---
+ecip: 1055
+title: Succinct PoW Using Merkle Mountain Ranges
+author: Zac Mitton @zmitton
+discussions-to: <https://github.com/ethereum/EIPs/pulls> (Pull Number TBD)
+status: Draft
+type: Standards Track
+category (*only required for Standard Track): Core
+created: 2019-03-12
+replaces (*optional): 210, 1094, 641
+
+---
+
+## Simple Summary
+
+The addition of a Merkle-Mountain-Range (MMR) root to blockheaders
+
+## Abstract
+<!--A short (~200 word) description of the technical issue being addressed.-->
+
+A block _chain_ could be interpreted as a linked-list using hashpointers from each block to its previous block. A Merkle-Mountain-Range is a type of tree that, will allow every block to point back to _all_ its previous blocks. This can be accomplished using only a single hashPointer in each block - the current MMR _root_. Adding this will allow proving the cummulative work of an entire chain in Olog(n) time, instead of the current O(n) time. In the case of ethereum, with small block-times, and over 7 million blocks and counting, this will open a ton of doors in the area light-clients, and data verification.
+
+
+
+## Motivation
+<!--The motivation is critical for EIPs that want to change the Ethereum protocol. It should clearly explain why the existing protocol specification is inadequate to address the problem that the EIP solves. EIP submissions without sufficient motivation may be rejected outright.-->
+The recent FlyClient [paper](https://eprint.iacr.org/2019/226.pdf) and [presentation](https://www.youtube.com/watch?v=BPNs9EVxWrA) at Scaling Bitcoin Represented a major breakthrough the ability to succinctly, and non-interactivly prove the cummulative work of a blockchain. 
```

Comment:
Second link could be more precise: https://youtu.be/BPNs9EVxWrA?t=8390

Reply:
```suggestion
The recent FlyClient [paper](https://eprint.iacr.org/2019/226.pdf) and [presentation](https://www.youtube.com/watch?v=BPNs9EVxWrA?t=8390) at Scaling Bitcoin Represented a major breakthrough the ability to succinctly, and non-interactivly prove the cumulative work of a blockchain.
```